tensorflow/inference/docker/2.4/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,131 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG PYTHON=python3.7
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
+ARG OPENSSL_VERSION=1.1.1g
+ARG TFS_SHORT_VERSION=2.4
+ARG TFS_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/serving/r2.4_aws/20210125-223221/cpu/py37/tensorflow_model_server
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV LD_LIBRARY_PATH='/usr/local/lib:$LD_LIBRARY_PATH'
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+ENV DEBIAN_FRONTEND=noninteractive
+
+# nginx + njs 
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    emacs \
+    git \
+    wget \
+    vim \
+    libbz2-dev \
+    liblzma-dev \
+    libffi-dev \
+    build-essential \
+    zlib1g-dev \
+    openssl \
```

Comment:
Is this needed? we install openssl from source (line 57).

Reply:
Install openssl and libssl to resolve https://ubuntu.com/security/CVE-2020-1971